lib/auth/auth_with_roles.go

```diff
@@ -3367,11 +3375,19 @@ func (a *ServerWithRoles) SetInstaller(ctx context.Context, inst types.Installer
 }
 
 // DeleteInstaller removes an installer script resource
-func (a *ServerWithRoles) DeleteInstaller(ctx context.Context) error {
+func (a *ServerWithRoles) DeleteInstaller(ctx context.Context, name string) error {
+	if err := a.action(apidefaults.Namespace, types.KindInstaller, types.VerbDelete); err != nil {
+		return trace.Wrap(err)
+	}
+	return trace.Wrap(a.authServer.DeleteInstaller(ctx, name))
+}
+
+// DeleteAllInstallers removes all installer script resources
+func (a *ServerWithRoles) DeleteAllInstallers(ctx context.Context) error {
 	if err := a.action(apidefaults.Namespace, types.KindInstaller, types.VerbDelete); err != nil {
```

Comment:
Include `VerbList` here as well.
```suggestion
	if err := a.action(apidefaults.Namespace, types.KindInstaller, types.VerbDelete, types.VerbList); err != nil {
```